### ISSUE TITLE
Release 1.1.0 Wishlist

### ISSUE THREAD

**Issue by ryanccarelli** [Dana-Farber-AIOS/pathml#150]
Here is a consolidated wishlist for next release:
- [ ] pathml.preprocessing
  - [x] Distributable MIF segmentation model (current model cannot be pickled) #130 
  - [ ] Stitch artifact correction #107 
  - [ ] Blur detection #44 
- [ ] pathml.ml
  - [ ] Design API
    - [ ] Support MIL and pixel labels
    - [ ] Wrap pytorch-imagemodels
    - [x] Optimized h5path dataloader #131
- [ ] pathml.datasets 
  - [ ] PESO #136 
- [ ] workflows
  - [x] pypi workflow #145 
  - [x] codecov workflow #162 
- [ ] pathml.core
  - [ ] slidedata.plot with masks, saving, etc
  - [ ] tile.plot

**Comment by jacobrosenthaldfci** [comment]
This is out of date now that we have released v2.0.0
